tests/test_user_management.py

```diff
@@ -1,19 +1,25 @@
 import json
 import unittest
+import pytest
 
 import requests
 
-from pylana.api import get_user_information
+from os import path
 
+from pylana.api import get_user_information
+from pylana import create_api
 
-class TestGetUserInformation(unittest.TestCase):
+@pytest.mark.skipif(not path.exists('./tests/admin_config.json'), 
+                    reason="admin privileges necessary")
```

Comment:
Here we require pytest explicitly for test code. We should change the readme accordingly. Currently, it still states that we unittest is enough to run the tests.

Also, I would make the message a bit more telling: "User management can only be tested with a user or system admin role.".